scripts/transpile-ts.js

```
@@ -0,0 +1,22 @@
+import * as ts from "typescript";
```

This import gives a webpack error, hence added a plugin to suppress that error